manala.deploy/tasks/strategy/git.yml

```diff
@@ -1,17 +1,23 @@
 ---
 
-- name: strategy/git > Cached repository
+- name: strategy/git > Update repository
   git:
     repo:           "{{ manala_deploy_strategy_git_repo }}"
-    dest:           "{{ deploy_helper.shared_path ~ '/cached-copy' }}"
+    dest:           "{{ manala_deploy_dir ~ '/repo' }}"
```

Comment:
You're absolutly right.
See: http://capistranorb.com/documentation/getting-started/configuration/
("repo_path")